srcopsmetrics/iterator.py

```diff
@@ -54,9 +54,9 @@ def __init__(
         self,
         *,
         entity_type: Optional[str] = None,
-        new_entities: Optional[str] = None,
-        accumulator: Optional[str] = None,
-        store_method: Optional[str] = None,
+        new_entities: List[Any] = None,
```

Comment:
I've generalized it into the `Iterable[Any]` - cause it can be in the end anything that is iterable. Then it depends on the implemented `store` function, how the iterated entity is processed